src/ApiRequestPanelElement.js

```diff
@@ -528,6 +534,7 @@ export class ApiRequestPanelElement extends EventsTargetMixin(LitElement) {
       .serverType="${serverType}"
       ?noServerSelector="${noServerSelector}"
       ?allowCustomBaseUri="${allowCustomBaseUri}"
+      credentialsSource="${credentialsSource}"
```

Comment:
Please, use the proper notation when passing a property and not an attribute. This should be `. credentialsSource`.